src/@next/Switch/SwitchStyle.tsx

```diff
@@ -54,9 +76,25 @@ export const InputStyle = styled.input`
     }
 
     &:before {
-      transform: translate(16px, -50%);
+      ${({ withIcon }: SwitchProps) => ({
+        transform: `translate(${withIcon ? '24px' : Spacing.space16} , -50%);`,
```

Comment:
same as above, give it a default transformation, and override with `&[data-with-icon]`

Reply:
done